verticapy/tests/utilities/test_utilities.py

```diff
@@ -496,8 +496,70 @@ def test_read_json(self):
         )
         assert result.shape() == (1782, 15)
         assert drop("v_temp_schema.titanic_verticapy_test_json", method="table")
-        # TODO
-        # test on archives
+        
+        # testing for multiple files
+        drop("public.laliga_verticapy_test_json", method="table")
+        vdf = read_json(
```

Comment:
comment this part of the test and write the following comment:
 - Not working: ERROR, Message: Failed to read json source []: Read error when expanding glob: ""it works locally""
Change the path to:
```python
path + '*.json'
```